storio-content-resolver-annotations-processor-test/build.gradle

```diff
@@ -0,0 +1,35 @@
+import org.gradle.internal.jvm.Jvm
+
+apply plugin: 'com.android.library'
+apply from: 'build.workaround-missing-resource.gradle'
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+    buildToolsVersion rootProject.ext.buildToolsVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+    }
+}
+
+dependencies {
+    testCompile libraries.storIOContentResolver
+    testCompile libraries.storIOContentResolverAnnotationsProcessor
+    testCompile libraries.googleTestingCompile
+    testCompile files(getRuntimeJar())
```

Comment:
well, technically it's a library, just a jar on your system. Point is to remove code duplication from build gradle, up2u though :)